js/utils/mutation_handlers/cached_handler.js

```diff
@@ -5,7 +5,7 @@ let lastEvent = {
 
 const MUTATION_STABILITY_TIMEOUT = 10; // milliseconds
 
-const isNotCached = timeStamp => title => {
+const isNotCached = (timeStamp, title) => {
```

Comment:
If I remember correctly we wanted all the events to have the same timestamp, although even with the currying the way `isNotCached` was being used did not achieve that. @JorikSchellekens can you shed more light on why the function was curried.

Reply:
The function was indeed just curried so that the timestamp could easily be made the same across events. The fact that it doesn't is a bug. isNotCached was written for arbitrary sequences of events with timestamps, however, that is probably an over-generalization. I'll modify the operation of this method in a separate issue and PR.